### DISCUSSION TITLE
Just Getting Started but have a problem

### DISCUSSION THREAD

**Discussion by krlanes** [aaronwmorris/indi-allsky#565] [General]
I really liked you project so I installed Astroberry and updated it and installed the project and rebooted. Everything seemed to go fine except when I went to https://astroberry.local:444/all-sky I get a login prompt which I put the user pi and password astroberry and I get Error 403. It wasn't clear to me in the setup if I should answer the user and password with a new user or an existing user. Now I see that this may have been a error. The astroberry default user is astroberry and not pi.

Thanks

Kurt

**Comment by aaronwmorris** [comment]
I believe the username is `admin` and the password is `secret`.

**Reply by krlanes** [reply]
OK I just tried that.  It also seems like my browser caches the username and password so I tried a different browser but no joy. What is the username and password used for during the init script?

Kurt

**Reply by aaronwmorris** [reply]
I will have to run some tests with Astroberry.  I recently removed the old web server based authentication so I am surprised you are still seeing it.  It is possible I just missed something related to the Astroberry configs.

**Comment by krlanes** [comment]
Thanks Aaron. Do you think I should try a fresh install? Is there a script to do that?

Thanks
Kurt

**Reply by aaronwmorris** [reply]
If I can fix the process, all you will need to do is re-run the setup.sh script.  That script is designed to be run repeatedly and correct issues in the install process.

**Comment by krlanes** [comment]
Aaron
Today I got the system working by installing the 64bit Ubuntu on a VirtualBox VM. I issued the commands to start the system e.g.
systemctl --user start indiserver
systemctl --user start indi-allsky
I was able to access the webpage without incident. When I select configure I open the webpage it prompts for a login. I answered the setup dialog with the same username and password as my login account but that is refused as invalid on the webpage.
Also I am unsure how to set up INDI. I tried to access it at 192.168.12.63:7624 but it does not come up.
Thanks
Kurt

**Comment by krlanes** [comment]
Ok I solved the logon issue. VirtualBox can get in a state where it inverts the caps lock and it was that way when I setup the password. I still have not solved the INDI problem.
